WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -29,6 +29,9 @@ class CompetitionsController < ApplicationController
 
   def new
     @competition = Competition.new
+    if current_user.any_kind_of_delegate?
+      @competition.delegates |= [current_user]
```

Comment:
Hi @jfly, I think you're right ActiveRecord must take care of this. If I clone a competition I'm already a delegate for I get the following values in delegate_ids, `value="6180,6181,6183,6180"` where I'm user 6180. But when I save the competition only the 3 unique records get stored in the database. So there are no real problems using << 

I think as this is cloning a competition we'd want to also clone the existing delegates rather than just set it to the current user.